package.json

```diff
@@ -13,7 +13,9 @@
     "docz:dev": "docz dev",
     "docz:build": "docz build",
     "build:css": "node-sass --output-style expanded --source-map true --source-map-contents true --precision 6 index.scss -o css/",
+    "build:icon": "yarn run clean:icon && mkdirp packages/de-code-icons/fonts && mkdirp packages/de-code-icons/css && cp icons/fonts/* packages/de-code-icons/fonts/ && cp icons/css/style.css packages/de-code-icons/css/de-code-icons.css",
```

Comment:
we need to minify the css and generate the source map file(if iconmoon is not giving the .map file)

https://github.com/twbs/bootstrap/blob/258a56b473edea843eb20e80bbc7bf49eb040a32/package.json#L24

Reply:
as of now, not required